### PR TITLE
Remove OpenXmlPackage.Close and references

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: samples/ThreadedCommentExample/Program.cs [hunk 1]
```diff
@@ -51,67 +51,65 @@ private static void Main(string[] args)
             }
 
             // WORKBOOK
-            SpreadsheetDocument? sd = ExampleUtilities.CreateSpreadsheetWorkbook(filePath, sheetName);
-
-            if (sd != null)
+            using (SpreadsheetDocument? sd = ExampleUtilities.CreateSpreadsheetWorkbook(filePath, sheetName))
```
Comment (twsouthwick) [inline samples/ThreadedCommentExample/Program.cs:54]: use `using SpreadsheetDocument? sd = ....;` instead without indenting the files

File: test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ public void CanCreatePresentationFromTemplate()
                 var part = packageDocument.PresentationPart;
                 var root = part.Presentation;
 
-                packageDocument.SaveAs(Path.GetTempFileName()).Close();
+                packageDocument.SaveAs(Path.GetTempFileName()).Dispose();
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs:23]: Can you refactor these usages to be within a using block?

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs [hunk 3]
```diff
@@ -33,7 +33,7 @@ public void CanCloneDocument()
                 {
                     var body = clone.MainDocumentPart.Document.Body;
                     body.InsertBefore(new Paragraph(new Run(new Text("Hello World"))), body.FirstChild);
-                    clone.SaveAs(file.Path).Close();
+                    clone.SaveAs(file.Path).Dispose();
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:36]: Please refactor things such that we never call dispose directly

**Review by twsouthwick — CHANGES_REQUESTED** [review]
A few comments about refactoring things so we never call dispose directly

File: samples/ThreadedCommentExample/Program.cs (hunk 1 above)
Comment (tomjebo) [reply samples/ThreadedCommentExample/Program.cs:54]: Sure

File: test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs (hunk 2 above)
Comment (tomjebo) [reply test/DocumentFormat.OpenXml.Tests/CreateFromTemplateTests.cs:23]: yes, of course

File: test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs (hunk 3 above)
Comment (tomjebo) [reply test/DocumentFormat.OpenXml.Tests/SaveAndCloneTests.cs:36]: yup

**Comment by tomjebo** [comment]
replaced this PR with #1373

**Pull request closed without merging**